open-machine-learning-jupyter-book/deep-learning/LSTM.md

```diff
@@ -13,4 +13,635 @@ kernelspec:
   name: python3
 ---
 
-# LSTM (TBD)
+# Long-Short Term Memory

```

Comment:
We only have the first letter as upper case in title.

Reply:
Done.